hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/legacy/core/jproto/JKey.java

```diff
@@ -256,9 +254,9 @@ private static JKey convertBasic(final com.hedera.hapi.node.base.Key key) throws
      *
      * @param jkey JKey object to be converted
      * @return the converted proto Key instance
-     * @throws DecoderException on an inconvertible given key
+     * @throws InvalidKeyException If the key is not a valid key type
      */
-    static Key convertJKeyBasic(JKey jkey) throws DecoderException {
+    static Key convertJKeyBasic(JKey jkey) throws InvalidKeyException {
```

Comment:
null annotations missing

Reply:
This is pre-existing code that has a ton of interactions elsewhere in mono service.  I am not going to add annotations or final keywords in this PR.  If we want to spend time cleaning up the to-be-removed mono service code, we can submit a separate issue.